packages/lodestar-db/src/schema.ts

```diff
@@ -35,6 +35,8 @@ export enum Bucket {
   validator = 16,
   lastProposedBlock = 17,
   proposedAttestations = 18,
+  // validator slashing protection
```

Comment:
bucket above is also used for same thing, so you can probably drop it?

Reply:
By dropping you mean using the `18` bucket? We should discuss how to handle changes in the DB. If we change the schema and users update it will break in a very nasty form and the db would have to be wiped out, all without any helpful message.

Reply:
I thought you are going to integrate slashing protection into Vc in this PR in which case you need to replace `proposedAttestations = 18` and it's usage with your own

Reply:
But the schema for the binary data stored in bucket 18 will be different. Once users run the client they will get a nasty SSZ decode error.

Reply:
True, we can just delete it a keep a skip bucket index, true

Reply:
Suggestion applied in https://github.com/ChainSafe/lodestar/pull/1674/commits/e879f745b0f2624bd0b34b4bac0edc0154627acc